app/src/swig/app.i

```diff
@@ -906,6 +907,12 @@ static firebase::AppOptions* AppOptionsLoadFromJsonConfig(const char* config) {
         // fire-(unity|mono)/<github-action-built|custom_built>
         RegisterLibraryInternal(
             libraryPrefix + "-buildsrc", Firebase.VersionInfo.BuildSource);
+        // On desktop, log a heartbeat after all Unity user agents have been
+        // registered.
+        if (!Firebase.Platform.PlatformInformation.IsAndroid
+            && !Firebase.Platform.PlatformInformation.IsIOS) {
```

Comment:
You don't need this since `LogHeartbeatInternal()` is no-op in those platform.  Also `IsIOS` does not include `tvOS`.

You can move the comment to `LogHeartbeatInternal()` below.

Reply:
Done. Also rephrased some comments.